src/es-module-shims.js

```diff
@@ -306,6 +306,12 @@ function resolveDeps (load, seen) {
       }
     }
 
+    // support progressive cycle binding updates
+    if (load.s) {
+      const fields = exports.filter(expt => expt.n !== 'default' && expt.ln).map(expt => `${expt.n}: ${expt.ln}`);
```

Comment:
Rather than `${expt.n}:` we should use the same form as the original exported identifier - `${source.slice(expt.s, expt.e)}` to also support string exports. We also don't actually need a default filter here since `expt.ln` is always guaranteed to be a valid identifier now:

```suggestion
      const fields = exports.filter(expt => expt.ln).map(expt => `${source.slice(expt.s, expt.e)}: ${expt.ln}`);
```

Reply:
Nice!